### DISCUSSION TITLE
Import Template from GitHub Repo

### DISCUSSION THREAD

**Discussion by TaprootFreak** [GrapesJS/grapesjs#5315] [Ideas]
Hi Together

I have successfully tried Grapes.js and find it very cool!

In order to use Grapes.js professionally, I need an option to import directly from a GitHub repo. 

<img width="897" alt="Bildschirmfoto 2023-08-17 um 16 05 32" src="https://github.com/GrapesJS/grapesjs/assets/142087526/43b1c0a1-1581-4aba-bf7b-01a2b077d6b3">

If there was a function to directly access a public GitHub repo, several people could work on a project at the same time. The CSS and HTML files on the GitHub repo would then always be the latest that everyone has agreed upon. 

I am looking forward to feedback

**Comment by lexoyo** [comment]
You need to create a backend app with oauth
Or front end with tokens may be possible
Take a look at the docs, the StorageManager config lets you specify a remote

**Reply by lexoyo** [reply]
I'm curious, what's your use case?
